docs/source/conf.py

```diff
@@ -51,6 +51,7 @@
     "construction_warning": False,
 }
 html_favicon = "https://www.gurobi.com/favicon.ico"
+html_static_path = ["_static"]
```

Comment:
If I remove it it compiles as expected. Do we need this? (I don't want to break the main docs website with this)

Reply:
Indeed! It seems to work without this line :)

Reply:
I was wrong here, we need this for the interactive plot!